charts/backstage/templates/route.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.route.enabled }}
+apiVersion: route.openshift.io/v1
+kind: Route
```

Comment:
we already have ingress...do we also need route? I think it's either one or the other. This route looks better as it had TLS.

Reply:
Leaving both, ingress is TLS due to the annotation: route.openshift.io/termination: "edge"